lib/LANraragi/Utils/Generic.pm

```diff
@@ -168,6 +168,26 @@ sub shasum {
     return $digest;
 }
 
+sub shasum_str {
```

Comment:
I think base `shasum` isn't used anywhere else with the switch to string reading for thumbnail hashes, so it could probably be removed.